backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -1059,6 +1060,29 @@ void convertToAoeObservation_pregnancyStatus_matchesJson() throws IOException {
     JSONAssert.assertEquals(expectedSerialized, actualSerialized, true);
   }
 
+  @Test
+  void convertToAoeObservation_genderofsexualpartners_matchesJson() throws IOException {
+    List<String> sexualPartners =
+        new ArrayList<>(
+            Arrays.asList("Transwoman", "Transman", "Nonbinary or gender non-conforming"));
```

Comment:
Here you can just do `List.of("Transwoman"` etc. Might also bet better to use the string constants rather the raw strings to protect against string drift

Reply:
updated